lib/index.js

```diff
@@ -340,10 +340,10 @@ module.exports = function systemJsTranslate(serverRoot, options) {
       var builderPromise;
 
       if (options.bundle) {
-        builderPromise = builder.bundle(fileName, options.buildFlags);
+        builderPromise = builder.bundle(fileName, extend(options.buildFlags, { config: { browserRootURL: 'http://' + req.headers.host } }));
       } else {
         // If source maps are wanted, also put in source files content
-        builderPromise = builder.compile(fileName, extend(options.buildFlags, { sourceMapContents: !!options.sourceMap }));
+        builderPromise = builder.compile(fileName, extend(options.buildFlags, { sourceMapContents: !!options.sourceMap, config: { browserRootURL: 'http://' + req.headers.host } }));
```

Comment:
Would be better to use `req.protocol + '://' + req.headers.host` (see http://expressjs.com/en/4x/api.html#req.protocol) as that would also support https (and pick up on `X-Forwarded-Proto` if the "trust proxy" setting is enabled).